modules/cluster-logging-deploy-cli.adoc

```diff
@@ -285,7 +285,6 @@ This default OpenShift Logging configuration should support a wide array of envi
 configuring {logging} components for information on modifications you can make to your OpenShift Logging cluster.
 ====
```

Comment:
Line 135 ,181-189 ,174, we are using deprecated version 5.1 here. Could we updated to 5.5.

Reply:
Line 327-330: we can use the new schema below in Logging 5.5 and later.  type can be fluentd or vector.

collection:
    type: "fluentd"  <10>

Reply:
@anpingli I wasn't sure what to change with the collection. It's currently listed as:

`collection:`
    `logs:`
      `type: "fluentd"  <10>`
      `fluentd: {}`

Reply:
```
collection:
    type: "fluentd"
```
collection
  logs:  (deprecate in future)
  fluentd: {}    (deprecate in future)